storm-core/src/jvm/backtype/storm/scheduler/resource/RAS_Node.java

```diff
@@ -193,17 +199,21 @@ public void freeAllSlots(Cluster cluster) {
     /**
      * Frees a single slot in this node
      * @param ws the slot to free
-     * @param cluster the cluster to update
      */
-    public void free(WorkerSlot ws, Cluster cluster) {
+    public void free(WorkerSlot ws) {
+        LOG.info("freeing ws {} on node {}", ws, _hostname);
```

Comment:
Can we call it 'slot' or something instead of 'ws' though. Until I read the type of the argument, I didn't know what ws was. Seeing that in the logs might be confusing.